### PR TITLE
install notes

### PR DESCRIPTION
**Pull request by nmorris**
issues with install were seen. wanted to capture solutions

### CODE REVIEW

**Comment by jdillworth** [comment]
This looks about right. As I recall I did have an .npmrc with a token before "npm login", so maybe my token was stale? Another account? I did recently change my npmjs.com password, so maybe that invalidated the token.

**Review by jdillworth — APPROVED** [review]
looks good

File: README.md [hunk 1]
```diff
@@ -25,6 +29,9 @@ yarn link
 # Go to the new app created from template.
 cd ../wireline-testing
 yarn link "@wirelineio/serverless-wireline"
+
+# now to install packages
```
Comment (richburdon) [inline README.md:33]: Nit: please make all comments sentences (i.e., start with capital letter, end with period). Also this doesn't need the "now to" part :).

File: README.md [hunk 2]
```diff
@@ -25,6 +29,9 @@ yarn link
 # Go to the new app created from template.
```
Comment (richburdon) [inline README.md:29]: This comment isn't required.

File: README.md [hunk 3]
```diff
@@ -11,7 +11,11 @@ https://www.npmjs.com/org/wirelineio
 yarn login
 ~~~~
 
-Check your auth token `~/.npmrc`.
+Check your auth token `~/.npmrc`. If not set or if there are issues downloading private packages
```
Comment (richburdon) [inline README.md:14]: Check `~/.npmrc` for auth token for private packages (i.e., via `yarn login`).

**Review by richburdon — APPROVED** [review]

**Comment by nmorris** [comment]
Readme has already been updated with better changes based on demo. closing

**Pull request closed without merging**